App-Service-Environment-AzFirewall/azuredeploy.json

```diff
@@ -0,0 +1,326 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
```

Comment:
As this template targets both Commercial and Azure Gov, should I add a value which would make this invalid in one of those environments (e.g. "East US" would break a deploy to Azure Gov)?

Reply:
no, the defaultValue for this param must be [resourceGroup().location] - so will work in all clouds